docs/index.md

```diff
@@ -3,174 +3,270 @@
+- `etcd_encryption` (Boolean) Encrypt etcd data.
+- `external_id` (String) Unique external identifier of the cluster.
+- `fips` (Boolean) Create cluster that uses FIPS Validated / Modules in Process cryptographic libraries
+- `host_prefix` (Number) Length of the prefix of the subnet assigned to each node.
+- `insecure` (Boolean) When set to 'true' enables insecure communication with the server. This disables verification of TLS certificates and host names and it isn't recommended for production environments.
+- `kms_key_arn` (String) The key ARN is the Amazon Resource Name (ARN) of a AWS KMS (Key Management Service) Key. It is a unique, fully qualified identifier for the AWS KMS Key. A key ARN includes the AWS account, Region, and the key ID.
+- `machine_cidr` (String) Block of IP addresses for nodes.
+- `max_replicas` (Number) Max replicas.
+- `min_replicas` (Number) Min replicas.
+- `multi_az` (Boolean) Indicates if the cluster should be deployed to multiple availability zones. Default value is 'false'.
+- `password` (String, Sensitive) User password.
+- `pod_cidr` (String) Block of IP addresses for pods.
+- `properties` (Map of String) User defined properties.
+- `proxy` (Attributes) proxy (see [below for nested schema](#nestedatt--proxy))
+- `replicas` (Number) Number of worker nodes to provision. Single zone clusters need at least 2 nodes, multizone clusters need at least 3 nodes.
+- `service_cidr` (String) Block of IP addresses for services.
+- `sts` (Attributes) STS Configuration (see [below for nested schema](#nestedatt--sts))
+- `tags` (Map of String) Apply user defined tags to all resources created in AWS.
+- `token` (String, Sensitive) Access or refresh token.
+- `token_url` (String) OpenID token URL.
+- `trusted_cas` (String) PEM encoded certificates of authorities that will be trusted. If this isn't explicitly specified then the provider will trust the certificate authorities trusted by default by the system.
+- `url` (String) URL of the OCM API server. This URL should point to your production or staging environment.
+- `user` (String) This input field is for your OCM user name.
+- `version` (String) Identifier of the version of OpenShift, for example 'openshift-v4.1.0'.
+
+### Read-Only
+
+- `api_url` (String) URL of the API server.
+- `ccs_enabled` (Boolean) Enables customer cloud subscription.
+- `console_url` (String) URL of the console.
+- `domain` (String) DNS Domain of Cluster
+- `id` (String) Unique identifier of the cluster.
+- `ocm_properties` (Map of String) Merged properties defined by OCM and the user defined 'properties'
+- `state` (String) State of the cluster.
+
+<a id="nestedatt--proxy"></a>
+### Nested Schema for `proxy`
+
+Optional:
+
+- `additional_trust_bundle` (String) a string contains contains a PEM-encoded X.509 certificate bundle that will be added to the nodes' trusted certificate store.
```

Comment:
Capitalize the first word of the sentence. I also think you can delete one of the "contains".